client/src/common/components/Form.jsx

```diff
@@ -16,20 +16,20 @@ class Form extends PureComponent {
     const {
       target: { value }
     } = event;
-    const { handleNameChange } = this.props;
+    const { onNameChange } = this.props;
 
     this.setState({ name: value });
-    handleNameChange(value);
+    onNameChange && onNameChange(value);
   };
 
   handleDescriptionChange = event => {
     const {
       target: { value }
     } = event;
-    const { handleDescriptionChange } = this.props;
+    const { onDescriptionChange } = this.props;
 
     this.setState({ description: value });
-    handleDescriptionChange(value);
+    onDescriptionChange && onDescriptionChange(value);
```

Comment:
thos one is required in props so no need to check it here - you'll get error about missing props during component creation